File: GdPicture/WinForm/C#/Document Clean Up/frmAbout.cs
using System;
using System.Windows.Forms;

namespace DocumentCleanUp
{
    public partial class frmAbout
    {
        public frmAbout()
        {
            InitializeComponent();
        }

        private void btnClose_Click(object sender, EventArgs e)
        {
            this.Close();
        }

        private void lblGdPictureWebsite_LinkClicked(object sender, LinkLabelLinkClickedEventArgs e)
        {
            System.Diagnostics.Process.Start("http://www.gdpicture.com/");
        }
    }
}

File: GdPicture/WinForm/vb.net/Image Processing/frmAbout.vb
Option Strict On
Option Explicit On

Public Class frmAbout

    Private Sub btnClose_Click(sender As Object, e As EventArgs) Handles btnClose.Click
        Me.Close()
    End Sub

    Private Sub lblGdPictureWebsite_LinkClicked(sender As Object, e As LinkLabelLinkClickedEventArgs) Handles lblGdPictureWebsite.LinkClicked
        Process.Start("http://www.gdpicture.com/")
    End Sub
End Class